maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -104,8 +112,23 @@ public int compare(Container o1, Container o2) {
 				if (failOnChanges) {
 					throw new MojoExecutionException("The runbundles have changed. Failing the build");
 				} else {
-					logger.warn("The runbundles have changed:");
+					logger.warn("The runbundles have changed");
 					run.setRunBundles(runBundles);
+					File runFile = run.getPropertiesFile();
+					BndEditModel bem = new BndEditModel(run.getWorkspace());
+					Document doc = new Document(IO.collect(runFile));
+					bem.loadFrom(doc);
+					List<VersionedClause> versionedClauses = new ArrayList<>();
```

Comment:
This seems to lose any ordering that may already exist in the original `-runbundles` statement.  See https://github.com/bndtools/bnd/blob/master/biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BndPlugin.groovy#L485-L492 where any original ordering is preserved in the updated statement.